src/buildHTML.js

```diff
@@ -55,14 +81,20 @@ const styleMap = {
  * a partial group (e.g. one created by \color). `surrounding` is an array
  * consisting type of nodes that will be added to the left and right.
  */
-export const buildExpression = function(expression, options, isRealGroup,
-        surrounding = [null, null]) {
+export const buildExpression = function(
+    expression: *[],
```

Comment:
Is it currently impossible to type `expression` and

Reply:
Sorry, sloppy me. I didn't go back through the stuff I'd already added types for way back when. Fixed.